### PR TITLE
Add route to form creation page; begin create_form.jsx cleanup

### PR DESCRIPTION
**Pull request by cmperez**

### CODE REVIEW

File: client/js/components/create_form.jsx [hunk 1]
```diff
@@ -24,9 +24,9 @@ var CreateForm = React.createClass ({
   createFormSchema: function(event) {
     event.preventDefault();
     var schemaName = AppStore.formTitle.split(" ").join("_");
-    var URL = "/api/schema" + schemaName;
+    var URL = "/api/schema/" + schemaName;
```
Comment (zindlerb) [inline client/js/components/create_form.jsx]: :pray:

**Comment by dduugg** [comment]
:+1: @rachelsison, you can fix trailing whitespace issues by installing the [EditorConfig](http://editorconfig.org/) plugin for your code editor. It makes future code reviews a lot easier.

File: client/js/router.jsx [hunk 2]
```diff
@@ -6,7 +6,8 @@ var Router = Backbone.Router.extend({
     signup: "signup",
     signin: "signin",
     "schema/:name": "dataPage",
-    user: "user"
+    user: "user",
+    create: "create"
```
Comment (dduugg) [inline client/js/router.jsx]: consider sorting the properties of this object

**Comment by cmperez** [comment]
fixed merge conflicts and sorted routes per dug's comment

**Comment by cmperez** [comment]
git issues have been worked out

**Comment by dduugg** [comment]
:+1:

**Comment by cmperez** [comment]
:gun: :gun:

**Pull request merged**